modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl

```diff
@@ -51,7 +51,7 @@
 			</#list>
 
 			<li>
-				<a class="dropdown-item" href="#1">
+				<a class="dropdown-item" href="https://customer.liferay.com/dxp-release-notes?p_p_id=com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet&_com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet_product=dxp&_com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet_tabs1=&_com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet_productVersion=7.4&_com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet_fromFixPackVersion=2024.101&_com_liferay_osb_customer_release_tool_web_portlet_ReleaseToolPortlet_toFixPackVersion=2024.105" target="_blank">
```

Comment:
Hardcoding this url feels off. Typically we should be able to dynamically set or configure things like text and urls to avoid needing to make commits for every content change. I am not sure of the context here, but is that an option for us?

Reply:
@ryanschuhler

Reply:
@ryanschuhler for the URL I applied it as I showed today, <#assign previousReleaseURL = "https://..." />